### PR TITLE
chore(website): add `yaml` & `graphql` to language support & sync table rows

### PR DESCRIPTION
**Pull request by afonsojramos**
## Summary

As discussed on Discord, we should, at least, have [YAML](https://github.com/biomejs/biome/issues/2365)
and [GraphQL](https://github.com/biomejs/biome/issues/1927) in the list of supported languages, so that users are aware that they might be supported in the future.

Took advantage to also sync up the changes across languages.

### CODE REVIEW

**Comment by nhedger** [comment]
I feel like there should be a link to the issue #2365 in the table.

**Comment by afonsojramos** [comment]
@NicholasLYang let me know if this looks good!

**Comment by ematipico** [comment]
> @NicholasLYang let me know if this looks good!

You tagged the wrong person 😅

**Review by Netail — APPROVED** [review]

**Review by ematipico — COMMENTED** [review]
Why so many changes? Is it possible your editor formatter did that? Technically, the only changes needed are two new rows in the table

**Comment by afonsojramos** [comment]
@ematipico Nope, no formatter, but clearly the last person did use a formatter and tried to align the table in markdown, but with long links it is a effort that will never produce perfect results, so I just edited to have 1 space between all table elements.

**Comment by afonsojramos** [comment]
@nhedger shall I migrate my changes to the new website repo?

**Comment by nhedger** [comment]
> @nhedger shall I migrate my changes to the new website repo?

I expect there will be other changes until the migration is complete, so let's merge here and when we're done, we'll sync everything.

Thanks for offering though!

**Comment by ematipico** [comment]
> @ematipico Nope, no formatter, but clearly the last person did use a formatter and tried to align the table in markdown, but with long links it is a effort that will never produce perfect results, so I just edited to have 1 space between all table elements.

It was me 👋🤣

**Comment by afonsojramos** [comment]
@ematipico so, do I need to apply any other changes? 
Is the preview deployment not working because of changes on this PR? The build ran without errors 🧐

**Comment by ematipico** [comment]
Of these new languages, only parsing is being worked. Formatting and linting haven't started yet. Could you please change it? We should copy the same info we have for HTML

**Comment by afonsojramos** [comment]
@ematipico should be good now!

File: website/src/content/docs/zh-cn/internals/language-support.mdx [hunk 1]
```diff
@@ -3,18 +3,22 @@ title: 语言支持
...
+| [Vue](#html-super-languages-support) | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
 | [Svelte](#html-super-languages-support) | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
-| CSS                                     | <span aria-label="正在进展中" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">⌛️</span> |
-| Markdown                                | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| [Astro](#html-super-languages-support) | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| CSS | <span aria-label="正在进展中" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">⌛️</span> |
```
Comment (unvalley) [inline website/src/content/docs/zh-cn/internals/language-support.mdx:18]:
```suggestion
| CSS | <span aria-label="支持" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="正在进展中" role="img">⌛️</span> |
```

File: website/src/content/docs/ja/internals/language-support.mdx [hunk 2]
```diff
@@ -3,18 +3,22 @@ title: 言語サポート
...
+| [Vue](#html-super-languages-support) | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
 | [Svelte](#html-super-languages-support) | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
-| CSS                                     | <span aria-label="進行中" role="img">✅️</span>         | <span aria-label="進行中" role="img">⌛️</span>      | <span aria-label="進行中ではない" role="img">⌛️</span> |
-| Markdown                                | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
+| [Astro](#html-super-languages-support) | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
+| CSS | <span aria-label="進行中" role="img">✅️</span> | <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">⌛️</span> |
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/language-support.mdx:18]:
```suggestion
| CSS | <span aria-label="対応済み" role="img">✅️</span> | <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">⌛️</span> |
```

File: website/src/content/docs/zh-cn/internals/language-support.mdx [hunk 3]
```diff
@@ -3,18 +3,22 @@ title: 语言支持
...
+| 语言 | 解析 | 格式化 | 检查 |
+| --- | ---- | ----- | --- |
+| [JavaScript](#javascript-support) | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| [TypeScript](#typescript-support) | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| JSX | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| TSX | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| JSON | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| JSONC | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> | <span aria-label="支持" role="img">✅</span> |
+| HTML | <span aria-label="不在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/zh-cn/internals/language-support.mdx:14]:
```suggestion
| HTML | <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```

File: website/src/content/docs/zh-cn/internals/language-support.mdx [hunk 4]
```diff
@@ -3,18 +3,22 @@ title: 语言支持
...
-| CSS                                     | <span aria-label="正在进展中" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">⌛️</span> |
-| Markdown                                | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| [Astro](#html-super-languages-support) | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| CSS | <span aria-label="正在进展中" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">⌛️</span> |
+| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="不在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="不在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/zh-cn/internals/language-support.mdx:20]:
```suggestion
| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```

**Comment by unvalley** [comment]
I've updated some aria-labels were wrong.

**Review by unvalley — APPROVED** [review]

File: website/src/content/docs/internals/language-support.mdx [hunk 5]
```diff
@@ -9,21 +9,22 @@ Legend:
...
-| Markdown                                | <span aria-label="Not in progress" role="img">🚫</span>     | <span aria-label="Not in Progress" role="img">🚫</span>     | <span aria-label="Not in Progress" role="img">🚫</span>     |
-
+| [Astro](#html-super-languages-support) | <span aria-label="Partially supported" role="img">⚠️</span> | <span aria-label="Partially supported" role="img">⚠️</span> | <span aria-label="Partially supported" role="img">⚠️</span> |
+| CSS | <span aria-label="Supported" role="img">✅️</span> | <span aria-label="In Progress" role="img">⌛️</span> | <span aria-label="In Progress" role="img">⌛️</span> |
+| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="In Progress" role="img">⌛️</span> | <span aria-label="Not in Progress" role="img">🚫</span> | <span aria-label="Not in Progress" role="img">🚫</span> |
+| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="In Progress" role="img">⌛️</span> | <span aria-label="Not in Progress" role="img">🚫</span> | <span aria-label="Not in Progress" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/internals/language-support.mdx:26]:
```suggestion
| [YAML](https://github.com/biomejs/biome/issues/2365) | <span aria-label="In Progress" role="img">⌛️</span> | <span aria-label="Not in Progress" role="img">🚫</span> | <span aria-label="Not in Progress" role="img">🚫</span> |
| [GraphQL](https://github.com/biomejs/biome/issues/1927) | <span aria-label="In Progress" role="img">⌛️</span> | <span aria-label="Not in Progress" role="img">🚫</span> | <span aria-label="Not in Progress" role="img">🚫</span> |
```

**Comment by unvalley** [comment]
Other languages are also should be fixed

File: website/src/content/docs/ja/internals/language-support.mdx [hunk 6]
```diff
@@ -3,18 +3,22 @@ title: 言語サポート
...
-| CSS                                     | <span aria-label="進行中" role="img">✅️</span>         | <span aria-label="進行中" role="img">⌛️</span>      | <span aria-label="進行中ではない" role="img">⌛️</span> |
-| Markdown                                | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
+| [Astro](#html-super-languages-support) | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
+| CSS | <span aria-label="対応済み" role="img">✅️</span> | <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中" role="img">⌛️</span> |
+| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
+| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/ja/internals/language-support.mdx:20]:
```suggestion
| [YAML](https://github.com/biomejs/biome/issues/2365) | <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
| [GraphQL](https://github.com/biomejs/biome/issues/1927) | <span aria-label="進行中" role="img">⌛️</span> | <span aria-label="進行中ではない" role="img">🚫</span> | <span aria-label="進行中ではない" role="img">🚫</span> |
```

File: website/src/content/docs/pt-br/internals/language-support.mdx [hunk 7]
```diff
@@ -3,18 +3,22 @@ title: Suporte de linguagens
...
-| Markdown                                      | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
+| [Astro](#suporte-a-super-linguagens-no-html) | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
+| CSS | <span aria-label="Disponível" role="img">✅️</span> | <span aria-label="Em progresso" role="img">⌛️</span> | <span aria-label="Não disponível" role="img">⌛️</span> |
+| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="Em progresso" role="img">⌛️</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
+| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="Em progresso" role="img">⌛️</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/pt-br/internals/language-support.mdx:20]:
```suggestion
| [YAML](https://github.com/biomejs/biome/issues/2365) | <span aria-label="Em progresso" role="img">⌛️</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
| [GraphQL](https://github.com/biomejs/biome/issues/1927) | <span aria-label="Em progresso" role="img">⌛️</span> | <span aria-label="Não disponível" role="img">🚫</span> | <span aria-label="Não disponível" role="img">🚫</span> |
```

File: website/src/content/docs/zh-cn/internals/language-support.mdx [hunk 8]
```diff
@@ -3,18 +3,22 @@ title: 语言支持
...
-| CSS                                     | <span aria-label="正在进展中" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">⌛️</span> |
-| Markdown                                | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| [Astro](#html-super-languages-support) | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| CSS | <span aria-label="支持" role="img">✅️</span> | <span aria-label="正在进展中" role="img">⌛️</span> | <span aria-label="正在进展中" role="img">⌛️</span> |
+| [YAML](https://github.com/biomejs/biome/issues/2365) <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
+| [GraphQL](https://github.com/biomejs/biome/issues/1927) <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```
Comment (unvalley) [inline website/src/content/docs/zh-cn/internals/language-support.mdx:20]:
```suggestion
| [YAML](https://github.com/biomejs/biome/issues/2365) | <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
| [GraphQL](https://github.com/biomejs/biome/issues/1927) | <span aria-label="正在进行中" role="img">⌛️</span> | <span aria-label="不在进行中" role="img">🚫</span> | <span aria-label="不在进行中" role="img">🚫</span> |
```

**Comment by afonsojramos** [comment]
Sorry, tried to do all of the changes with regex (always good to practice), but ended up not being entirely successful 😅

**Pull request merged**